src/utils/waypoint.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react"; // { useState , FormEventHandler },
+import { useSelector, useDispatch } from "react-redux";
+import moment from "moment";
+
+import type { Waypoint } from "@bindings/protobufs/Waypoint";
+
+import {
+  selectActiveWaypoint,
+  selectIsWaypointEdit,
+} from "@features/device/deviceSelectors";
+import { requestNewWaypoint } from "@features/device/deviceActions";
+import { deviceSliceActions } from "@features/device/deviceSlice";
+
+// This file contains utils for: Delete waypoint and Toggle editWaypoint
+
+// Delete waypoint
+const useDeleteWaypoint = () => {
```

Comment:
Is this a hook? If not, don't call it `use...`

https://reactjs.org/docs/hooks-custom.html#extracting-a-custom-hook